.github/dockerfiles/ov_build/ubuntu_22_04_riscv/Dockerfile

```diff
@@ -10,7 +10,9 @@ RUN echo 'Acquire::Retries "10";' > /etc/apt/apt.conf && \
     echo 'APT::Get::no-install-recommends "true";' >> /etc/apt/apt.conf
 
 ENV DEBIAN_FRONTEND="noninteractive" \
-    TZ="Europe/London"
+    TZ="Europe/London" \
+    CCACHE_MAXSIZE=2G \
+    RISCV_TOOLCHAIN_ROOT=/__w/openvino/xuantie/Xuantie-900-gcc-linux-5.15.0-glibc-x86_64-V2.8.1/
```

Comment:
__w/openvino - specific runner folder, we need to avoid hardcoding it in the docker